README.md

```diff
@@ -12,7 +12,7 @@ Important notice: In order to guarantee compatibility between S64 DA and s64da-b
 - Install additional packages, for Python 3.6 eg. with: `/usr/bin/python3.6 -m pip install -r requirements.txt`
 - The `psql` PostgreSQL client
 - For loading the data, the database must be accessible with the user
-  `postgres` *without password*
+  `postgres` or `enterprisedb` *without password*
```

Comment:
Could you change the summary as well to mention that it now works with EnterpriseDB as well? Otherwise the sudden mentions of EDB might be a bit surprising to the reader.

Reply:
Correct. Done.